contrib/examples/apiserver/instance.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
   name: test-instance
   namespace: test-ns
 spec:
-  externalClusterServiceClassName: test-serviceclass
-  externalClusterServicePlanName: example-plan-1
+  clusterServiceClassName: test-serviceclass
```

Comment:
Since we're reducing users typing, since these are inside ClusterServiceInstance, is it enough to have them just be:
servicePlanName
serviceClassName